lms/static/certificates/sass/_print.scss

```diff
@@ -94,6 +94,12 @@
 // ------------------------------
 @media print {
 
+    html,
+    body {
+        height: 650px;
+        page-break-after: avoid;
```

Comment:
If this is not needed, please remove. If it is still valid, can you add an inline comment noting what it does?